package.json

```diff
@@ -41,7 +41,7 @@
   },
   "dependencies": {
     "@fontsource/inter": "^5.0.18",
-    "antd": "5.9.2",
+    "antd": "5.13.0",
```

Comment:
I need this update for semantic footer styling  https://ant.design/components/drawer#semantic-dom